_scss/_release-notes.scss

```diff
@@ -0,0 +1,54 @@
+.release-date {
```

Comment:
I think there's work we could do in this file to keep things closer to the design system, but I think it's probably most efficient to let this in as it stands and then include it in a docs-wide theming review in the future.

Reply:
Awesome! Let me know if you need anything!